CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## Fleet 4.17.1 (Jul 22, 2022)
+
+* Fixed a bug causing an error when converting users to SSO login. 
+
+* Fixed a bug causing the **Edit User** modal to hang when editing multiple users.
+
+* Fixed a bug that caused Ububtu hosts to display an inaccurate OS version. 
+
+* Fixed a bug affecting exporting live query results.
+
+* Fixed a bug in the Fleet UI affection live query result counts.
```

Comment:
```suggestion
* Fixed a bug in the Fleet UI affecting live query result counts.
```

Reply:
@zwass 
Great question! This is how I remember the correct usage.
"Affecting" is correct. Think of the **A** in **affect** as also standing for **action** words or verbs. The **E** in **effect** can also stand for **end** result so it tends to be used as a noun. Affect can be used as a noun when describing emotions.